capstone/fitness/src/main/java/com/google/sps/servlets/Utils.java

```diff
@@ -0,0 +1,63 @@
+package com.google.sps.servlets;
+
+import javax.servlet.http.HttpServletRequest;
+import com.google.api.client.auth.oauth2.AuthorizationCodeFlow;
+import java.io.IOException;
+import javax.servlet.ServletException;
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeFlow;
+
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import java.util.*;
+
+import com.google.api.client.extensions.appengine.datastore.AppEngineDataStoreFactory;
+import com.google.api.client.extensions.appengine.http.UrlFetchTransport;
+
+import com.google.api.services.calendar.CalendarScopes;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.googleapis.auth.oauth2.GoogleClientSecrets;
+
+import com.google.api.client.util.Preconditions;
+import java.io.InputStreamReader;
+
+class Utils {
```

Comment:
Per your comment for this PR, wondering if we should change this to AuthorizationUtils or something more descriptive.

Reply:
I think that would make sense.